packages/playwright/src/index.ts

```diff
@@ -147,7 +147,18 @@ export default class AxeBuilder {
       context
     );
     const partials = await partialResults.getPartials();
-    return this.finishRun(partials);
+
+    try {
+      return this.finishRun(partials);
+    } catch (error) {
+      throw new Error(
+        `Error: ${JSON.stringify(
+          error,
+          null,
+          2
+        )}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
```

Comment:
I don't think this does what you expect it to.

```
throw new Error(`Error: ${JSON.stringify(new Error("boom"), null, 2)}\n Please ...`)
VM335:1 Uncaught Error: Error: {}
 Please ...
    at <anonymous>:1:7
```

Note `Error: {}`, not `Error: boom`.

Reply:
Thoughts on simply throwing the `error` and providing the link to the docs. 


```
   try {
      return this.finishRun(partials);
    } catch (error) {
      throw new Error(
        `Error: ${error}\n Please check out https://github.com/dequelabs/axe-core-npm/blob/develop/packages/playwright/error-handling.md`
      );
    }
 ```